beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1104,6 +1173,17 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             .map(Some)
     }
 
+    /// Returns the blobs at the given root, if any.
+    ///
+    /// ## Errors
+    /// May return a database error.
+    pub fn get_blobs(&self, block_root: &Hash256) -> Result<BlobSidecarList<T::EthSpec>, Error> {
```

Comment:
Yes, so this originally returned an `Option`, but we removed it in this PR:  https://github.com/sigp/lighthouse/pull/4535

It seemed like the place where the `Option` was primarily useful was in the beacon API because for blocks it dictates whether we should return a 404, but for the blobs endpoint, the 404 is determined by whether we have the corresponding block (not blobs) so the `Option` here didn't end up useful.

Wouldn't be opposed to adding it back, but it has been considered :)